storage/elasticsearch/elasticsearch.go

```diff
@@ -101,8 +99,7 @@ type ElasticSearchStorage struct {
 }
 
 func (c *ElasticSearchStorage) StoreFlows(flows []*flow.Flow) error {
-	c.indexer.Start()
-	defer c.indexer.Stop()
+	defer c.indexer.Flush()
```

Comment:
It's a bad idea to call flush on all flows() .... resulting is the same as synchronous call
This code should me under indexer.Start()/Stop() like before

indexer.Flush() should be called on the ElasticSearchStorage.Close()